tests/no_results_test.go

```diff
@@ -0,0 +1,130 @@
+package tests
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	qt "github.com/frankban/quicktest"
+	"github.com/vocdoni/davinci-node/circuits"
+	"github.com/vocdoni/davinci-node/types"
+	"github.com/vocdoni/davinci-node/workers"
+)
+
+func TestNoResults(t *testing.T) {
```

Comment:
Does this test load the full sequencer again? If so, we should unify this test with the integration test or find a way to execute multiple tests while keeping the same circuit artifacts in memory. 

The must is that we do only load the proving keys once for the whole tests/ package.

Reply:
this test should be deleted, it was just for reproduce the error